esphome/components/seeed_mr60fda2/seeed_mr60fda2.cpp

```diff
@@ -0,0 +1,401 @@
+        this->fall_detected_binary_sensor_->publish_state(this->current_frame_buf_[LEN_TO_HEAD_CKSUM]);
+      }
+      this->current_frame_locate_ = LOCATE_FRAME_HEADER;
+      break;
+    case PEOPLE_EXIST_TYPE_BUFFER:
+      if (this->people_exist_binary_sensor_ != nullptr)
+        this->people_exist_binary_sensor_->publish_state(this->current_frame_buf_[LEN_TO_HEAD_CKSUM]);
+      this->current_frame_locate_ = LOCATE_FRAME_HEADER;
+      break;
+    case RESULT_INSTALL_HEIGHT:
+      if (this->current_data_buf_[0]) {
+        ESP_LOGD(TAG, "Successfully set the mounting height");
+      } else {
+        ESP_LOGD(TAG, "Failed to set the mounting height");
+      }
+      this->current_frame_locate_ = LOCATE_FRAME_HEADER;
+      break;
+    case RESULT_HEIGHT_THRESHOLD:
+      if (this->current_data_buf_[0]) {
+        ESP_LOGD(TAG, "Successfully set the height threshold");
+      } else {
+        ESP_LOGD(TAG, "Failed to set the height threshold");
+      }
+      this->current_frame_locate_ = LOCATE_FRAME_HEADER;
+      break;
+    case RESULT_SENSITIVITY:
+      if (this->current_data_buf_[0]) {
+        ESP_LOGD(TAG, "Successfully set the sensitivity");
+      } else {
+        ESP_LOGD(TAG, "Failed to set the sensitivity");
+      }
+      this->current_frame_locate_ = LOCATE_FRAME_HEADER;
+      break;
+    case RESULT_PARAMETERS:
+      // ESP_LOGD(
+      //     TAG,
+      //     "GET CURRENT_FRAME: 0x%02x 0x%02x 0x%02x 0x%02x, 0x%02x 0x%02x 0x%02x 0x%02x, 0x%02x 0x%02x 0x%02x 0x%02x",
+      //     this->current_frame_buf_[8], this->current_frame_buf_[9], this->current_frame_buf_[10],
+      //     this->current_frame_buf_[11], this->current_frame_buf_[12], this->current_frame_buf_[13],
+      //     this->current_frame_buf_[14], this->current_frame_buf_[15], this->current_frame_buf_[16],
+      //     this->current_frame_buf_[17], this->current_frame_buf_[18], this->current_frame_buf_[19]);
+      // ESP_LOGD(
+      //     TAG,
+      //     "GET CURRENT_FRAME_2: 0x%02x 0x%02x 0x%02x 0x%02x, 0x%02x 0x%02x 0x%02x 0x%02x, 0x%02x 0x%02x 0x%02x
+      //     0x%02x", this->current_data_buf_[0], this->current_data_buf_[1], this->current_data_buf_[2],
+      //     this->current_data_buf_[3], this->current_data_buf_[4], this->current_data_buf_[5],
+      //     this->current_data_buf_[6], this->current_data_buf_[7], this->current_data_buf_[8],
+      //     this->current_data_buf_[9], this->current_data_buf_[10], this->current_data_buf_[11]);
+      this->current_install_height_int_ =
+          encode_uint32(current_data_buf_[3], current_data_buf_[2], current_data_buf_[1], current_data_buf_[0]);
+      float install_height_float;
+      memcpy(&install_height_float, &current_install_height_int_, sizeof(float));
+      select_index_ = find_nearest_index(install_height_float, INSTALL_HEIGHT, 7);
+      this->install_height_select_->publish_state(this->install_height_select_->at(select_index_).value());
```

Comment:
You need to check these are not nullptr otherwise if a user doesnt configure it, it will crash
```suggestion
      if (this->install_height_select_ != nullptr) {
        this->current_install_height_int_ =
          encode_uint32(current_data_buf_[3], current_data_buf_[2], current_data_buf_[1], current_data_buf_[0]);
        float install_height_float;
        memcpy(&install_height_float, &current_install_height_int_, sizeof(float));
        select_index_ = find_nearest_index(install_height_float, INSTALL_HEIGHT, 7);
        this->install_height_select_->publish_state(this->install_height_select_->at(select_index_).value());
      }
```